R/explanation.R

```diff
@@ -332,3 +336,21 @@ get_list_approaches <- function(n_features, approach) {
   }
   return(l)
 }
+
+#' @keywords internal
+explainer_x_test <- function(x_test, feature_labels) {
+
+  if (ncol(x_test) == length(feature_labels)) {
+
+    if (all.equal(colnames(x_test), feature_labels))
```

Comment:
```suggestion
    if (identical(colnames(x_test), feature_labels))
```
all.equal does not give FALSE when they are different, but rather how they are different. Identical always produce TRUE/FALSE